src/functions/color.js

```diff
@@ -86,3 +87,33 @@ defineFunction({
     htmlBuilder,
     mathmlBuilder,
 });
+
+defineFunction({
+    type: "color",
+    names: ["\\color"],
+    props: {
+        numArgs: 1,
+        allowedInText: true,
+        greediness: 3,
+        argTypes: ["color"],
+    },
+    handler(context, args) {
+        const {parser, breakOnTokenText} = context;
+
+        const color = args[0];
```

Comment:
This was unexpectedly nice, the parser has already parsed the arguments for us, yay!